common/config/rush/pnpm-lock.yaml

```diff
@@ -8154,24 +7669,6 @@ packages:
       node: '>=6.0'
     resolution:
       integrity: sha512-lMo8DJ8u6JRWp0/Y4XLa/atVDr75H9litKlb2E5j3V3MesoL50EBgZDWoLT3F/LztVnG67GjPXLZpqcky/UMnQ==
-  /xmlbuilder/9.0.7:
-    dev: false
-    engines:
-      node: '>=4.0'
-    resolution:
-      integrity: sha1-Ey7mPS7FVlxVfiD0wi35rKaGsQ0=
-  /xmldom/0.6.0:
```

Comment:
this is a vulnerable dep that will be removed from our deps list when we remove event-processor-host. cc @HarshaNalluru